src/main/java/TTT/controller/SecurityController.java

```diff
@@ -27,10 +27,17 @@ public String getRegisterPage(Model model){
     @PostMapping("/register")
     public String registerUser(CustomUser customUser, BindingResult bindingResult){
 
-        if (bindingResult.hasErrors()){
+
+        // if has errors, return customUser to register form
+        if (bindingResult.hasErrors()) {
             return "security/register";
         }
+        if (customUser.getPassword().length() < 8){
+            return "index";
+        }
```

Comment:
Let's discuss how to validate password on next meeting

Reply:
ok...